apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -8,21 +8,39 @@ const examplePrompts = [
     id: 'random_mc',
     question:
       'Write a multiple choice question asking the user to choose the median of 5 random numbers between 1 and 100. Display all of the numbers to the user, and then make each of the numbers a potential answer, with the median being the correct answer.',
+    userPrompt:
+      'Each random number generated should be a potential answer to the multiple-choice question. Randomize the order of the numbers.',
+    prompt:
+      'Write a multiple choice question asking the user to choose the median of 5 random numbers between 1 and 100. Display all numbers to the user, and ask them to choose the median.',
+    gradingPrompt: 'Make the actual median the correct answer.',
   },
   {
     id: 'random_int',
     question:
       'Write a question that asks the user to multiply two integers. You should randomly generate two integers A and B, display them to the user, and then ask the user to provide the product C = A * B in an integer input box. The correct answer should be the product of the two numbers that you generated.',
+    prompt:
+      'Write a question that asks the user to multiply two integers. You should randomly generate two integers A and B, display them to the user, and then ask the user to provide the product C = A * B.',
+    userPrompt: 'Provide an integer input box for the user to enter the product.',
+    gradingPrompt: 'The correct answer is the actual product',
   },
   {
     id: 'basic_int',
     question:
       'Write a question asking "What The Answer to the Ultimate Question of Life, the Universe, and Everything?". Provide an integer box for the user to answer. The correct answer is 42.',
+    prompt:
+      'Write a question asking "What The Answer to the Ultimate Question of Life, the Universe, and Everything?".',
```

Comment:
Is "What The Answer..." the canonical phrasing from Hitchhiker's Guide, or should this actually be "What is The Answer..."?

Reply:
Changed in 09ebd145316f730bdcbac5525887fda484b63483